<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contenido_plots_costos</class>
 <widget class="QWidget" name="contenido_plots_costos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox1">
       <property name="title">
        <string>Method:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QRadioButton" name="uniform">
          <property name="text">
           <string>Uniform</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Ignored</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Ignored</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="diverse">
          <property name="text">
           <string>Diverse</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="0" colspan="3">
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox4">
         <property name="title">
          <string>Cost information</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="1" column="0" colspan="3">
           <widget class="QTableWidget" name="capitalcost">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>650</width>
              <height>118</height>
             </size>
            </property>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <column>
             <property name="text">
              <string/>
             </property>
            </column>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="labelopera">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Operation cost</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Units</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBoxUOP">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="labelcapital">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Capital cost:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBoxCapital">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Select one...</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>a + b*A^c</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>b*A^c</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QTableWidget" name="operationcost">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>1400</width>
              <height>80</height>
             </size>
            </property>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <column>
             <property name="text">
              <string/>
             </property>
            </column>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QLabel" name="label_WARNIG1">
            <property name="maximumSize">
             <size>
              <width>650</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;a name=&quot;tw-target-text&quot;/&gt;&lt;span style=&quot; font-family:'inherit'; font-size:7pt; font-style:italic; color:#212121;&quot;&gt;T&lt;/span&gt;&lt;span style=&quot; font-family:'inherit'; font-size:7pt; font-style:italic; color:#212121;&quot;&gt;he value of the constants will depend on the unit system selected in the workspace, in case of working with english system the value of these will be&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-family:'inherit'; font-size:7pt; font-style:italic; color:#212121;&quot;&gt;for area in ft&lt;/span&gt;&lt;span style=&quot; font-family:'inherit'; font-size:7pt; font-style:italic; color:#212121; vertical-align:super;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-family:'inherit'; font-size:7pt; font-style:italic; color:#212121;&quot;&gt; and for the international system in mt&lt;/span&gt;&lt;span style=&quot; font-family:'inherit'; font-size:7pt; font-style:italic; color:#212121; vertical-align:super;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-family:'inherit'; font-size:7pt; font-style:italic; color:#212121;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="margin">
             <number>6</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Inputs</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="2">
         <widget class="QPushButton" name="help">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/resources/Resources/ayuda.png</normaloff>:/resources/Resources/ayuda.png</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelinc">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Increment:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelmin">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Minimiun:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxK">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxMax">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxMin">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelk">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>K:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxInc">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelmax">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Maximun:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QGroupBox" name="groupBox3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Auxiliary services:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="2" rowspan="4">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="labelAux">
            <property name="text">
             <string>Auxiliary services</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTableWidget" name="Services">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>130</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>105</height>
             </size>
            </property>
            <row>
             <property name="text">
              <string>Heating Service</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Cooling Service</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Tsupply</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ttarget</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>h</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>Empty</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>Empty</string>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string>Empty</string>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>Empty</string>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string>Empty</string>
             </property>
            </item>
            <item row="1" column="2">
             <property name="text">
              <string>Empty</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="TypeDesign">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>23</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Select one...</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Countercurrent</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cococurrent</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelfilm">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Film coeficcient:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxFilm">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labeltemp">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Temperature:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelunits">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Units:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="comboBoxTemp">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelType">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Type of design:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
